debian/marquee/nginx/conf/http.conf

```diff
@@ -0,0 +1,6 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    server_name _;
+    return 301 https://$host$request_uri;
```

Comment:
To make the webroot certbot thing work, we'll need:
```
  location ^~ /.well-known/acme-challenge/ {
    default_type application/jose+json;
  }

  location / {
    return 301 https://$host$request_uri;
  }
```

Reply:
Wait, we don't redirect other URLs than `/` or does this mean something else?

Reply:
> To make the webroot certbot thing work, we'll need:
> ```
>  location ^~ /.well-known/acme-challenge/ {
>    default_type application/jose+json;
>  }
>
>  location / {
>    return 301 https://$host$request_uri;
>  }
> ```

OK, added